src/mame/includes/p2000t.h

```diff
@@ -27,16 +28,19 @@ class p2000t_state : public driver_device
 		, m_maincpu(*this, "maincpu")
 		, m_speaker(*this, "speaker")
 		, m_mdcr(*this, "mdcr")
+		, m_ram(*this, RAM_TAG)
 		, m_keyboard(*this, "KEY.%u", 0)
 	{
 	}
 
 	void p2000t(machine_config &config);
+	void init_p2000();
```

Comment:
Is there a specific reason you implemented this as an init function, rather than overriding `machine_start`?  We tend to discourage init functions for things other than unscrambling/decrypting ROMs or installing patches to work around emulation issues.

Reply:
No, taken from zx spectrum example. Will fix.